src/schemas/json/aiproj-1.1.json

```
@@ -0,0 +1,502 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Settings for project analysis by the Application Inspector",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "Version": {
+      "type": "string",
+      "enum": ["1.0", "1.1"]
+    },
+    "BlackBoxSettings": {
+      "type": ["object", "null"],
+      "description": "Black box and autocheck options",
```

```suggestion
      "description": "Black box and autocheck options\n",
```

Optional fix: link to docs can be added after trailing new line.

---

thank you for this comment, but we haven't got any docs that could be wrote there(